core/string/translation.cpp

```diff
@@ -456,7 +458,7 @@ String TranslationServer::get_locale_name(const String &p_locale) const {
 		}
 	}
 
-	String name = language_map[lang_name];
+	String name = p_native ? native_language_map[lang_name] : language_map[lang_name];
```

Comment:
`get_native_language_name` falls back to English but `get_native_locale_name` (this one) does not. Feels inconsistent to me.